make/autoconf/flags-cflags.m4

```diff
@@ -152,8 +152,6 @@ AC_DEFUN([FLAGS_SETUP_DEBUG_SYMBOLS],
 
     CFLAGS_DEBUG_SYMBOLS="-g ${GDWARF_FLAGS}"
     ASFLAGS_DEBUG_SYMBOLS="-g"
-  elif test "x$TOOLCHAIN_TYPE" = xxlc; then
-    CFLAGS_DEBUG_SYMBOLS="-g1"
```

Comment:
This and other parts in support of xlc need to stay: We still use xlc on AIX.

Reply:
Restored